src/OpenFeature.SDK/Model/Value.cs

```diff
@@ -17,6 +17,27 @@ public class Value
         /// </summary>
         public Value() => this._innerValue = null;
 
+        /// <summary>
+        /// Creates a Value with the inner set to the object
+        /// </summary>
+        /// <param name="value"><see cref="Object">The object to set as the inner value</see></param>
+        public Value(Object value)
+        {
+            // integer is a special case, convert those.
+            this._innerValue = value is int ? Convert.ToDouble(value) : value;
+            if (!(this.IsNull()

```

Comment:
Just an off-hand comment, but I wonder if all these is/as methods should be properties?

Reply:
I'd be totally ok with that. I'm comfortable with C#, but use it infrequently enough that I don't have a good feel for what's idiomatic in terms of such stylistic choices.

If you think properties are a better choice I'll happily switch. I'd much rather do so now when we're already doing breaking changes if you think it's a better choice.

Reply:
I would say that properties would be more idiomatic. Generally if it could be thought of as data, and you are getting it, and it doesn't cause a bunch of side-effects, then properties are a good choice.

I didn't notice until I was using it and it felt a bit off.

MS has some more specific guidelines around it.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/property

Reply:
Will make this change. Thanks again!